pandas/core/generic.py

```diff
@@ -244,7 +271,7 @@ def _validate_dtype(self, dtype):
     # Construction
 
     @property
-    def _constructor(self):
+    def _constructor(self: FrameOrSeries) -> FrameOrSeries:
```

Comment:
should the return type by Type[FrameOrSeries]?

Reply:
Good catch.

needed a few ignores though!